### ISSUE TITLE
installation problem

### ISSUE THREAD

**Issue by mohseniaref** [pyrocko/pyrocko#224]
Dear pyrocko community

I face this error when I am installing pyroko on Centos 6.9 .Would you please help me to resolve this problem?
```
running install
running build
running build_py
creating build/lib.linux-x86_64-2.7
creating build/lib.linux-x86_64-2.7/pyrocko
copying src/css.py -> build/lib.linux-x86_64-2.7/pyrocko
copying src/moment_tensor_viewer.py -> build/lib.linux-x86_64-2.7/pyrocko
copying src/file.py -> build/lib.linux-x86_64-2.7/pyrocko
copying src/gse2_io_wrap.py -> build/lib.linux-x86_64-2.7/pyrocko
copying src/parimap.py -> build/lib.linux-x86_64-2.7/pyrocko
copying src/pile.py -> build/lib.linux-x86_64-2.7/pyrocko
copying src/io.py -> build/lib.linux-x86_64-2.7/pyrocko
copying src/beachball.py -> build/lib.linux-x86_64-2.7/pyrocko
copying src/crustdb.py -> build/lib.linux-x86_64-2.7/pyrocko
copying src/seisan_waveform.py -> build/lib.linux-x86_64-2.7/pyrocko
copying src/ims.py -> build/lib.linux-x86_64-2.7/pyrocko
copying src/iris_ws.py -> build/lib.linux-x86_64-2.7/pyrocko
copying src/pz.py -> build/lib.linux-x86_64-2.7/pyrocko
copying src/segy.py -> build/lib.linux-x86_64-2.7/pyrocko
copying src/hudson.py -> build/lib.linux-x86_64-2.7/pyrocko
copying src/crust2x2_data.py -> build/lib.linux-x86_64-2.7/pyrocko
copying src/serial_hamster.py -> build/lib.linux-x86_64-2.7/pyrocko
copying src/evalresp.py -> build/lib.linux-x86_64-2.7/pyrocko
copying src/marker.py -> build/lib.linux-x86_64-2.7/pyrocko
copying src/kan.py -> build/lib.linux-x86_64-2.7/pyrocko
copying src/util.py -> build/lib.linux-x86_64-2.7/pyrocko
copying src/response_plot.py -> build/lib.linux-x86_64-2.7/pyrocko
copying src/slink.py -> build/lib.linux-x86_64-2.7/pyrocko
copying src/orthodrome.py -> build/lib.linux-x86_64-2.7/pyrocko
copying src/tectonics.py -> build/lib.linux-x86_64-2.7/pyrocko
copying src/geonames.py -> build/lib.linux-x86_64-2.7/pyrocko
copying src/snuffling.py -> build/lib.linux-x86_64-2.7/pyrocko
copying src/plot.py -> build/lib.linux-x86_64-2.7/pyrocko
copying src/shadow_pile.py -> build/lib.linux-x86_64-2.7/pyrocko
copying src/crustdb_abbr.py -> build/lib.linux-x86_64-2.7/pyrocko
copying src/guts.py -> build/lib.linux-x86_64-2.7/pyrocko
copying src/ahfullgreen.py -> build/lib.linux-x86_64-2.7/pyrocko
copying src/dummy_progressbar.py -> build/lib.linux-x86_64-2.7/pyrocko
copying src/guts_array.py -> build/lib.linux-x86_64-2.7/pyrocko
copying src/mseed.py -> build/lib.linux-x86_64-2.7/pyrocko
copying src/suds.py -> build/lib.linux-x86_64-2.7/pyrocko
copying src/__init__.py -> build/lib.linux-x86_64-2.7/pyrocko
copying src/info.py -> build/lib.linux-x86_64-2.7/pyrocko
copying src/cake_plot.py -> build/lib.linux-x86_64-2.7/pyrocko
copying src/trace.py -> build/lib.linux-x86_64-2.7/pyrocko
copying src/gcf.py -> build/lib.linux-x86_64-2.7/pyrocko
copying src/snuffler.py -> build/lib.linux-x86_64-2.7/pyrocko
copying src/gse1.py -> build/lib.linux-x86_64-2.7/pyrocko
copying src/yaff.py -> build/lib.linux-x86_64-2.7/pyrocko
copying src/datacube.py -> build/lib.linux-x86_64-2.7/pyrocko
copying src/pchain.py -> build/lib.linux-x86_64-2.7/pyrocko
copying src/rdseed.py -> build/lib.linux-x86_64-2.7/pyrocko
copying src/seed.py -> build/lib.linux-x86_64-2.7/pyrocko
copying src/pile_viewer.py -> build/lib.linux-x86_64-2.7/pyrocko
copying src/io_common.py -> build/lib.linux-x86_64-2.7/pyrocko
copying src/model.py -> build/lib.linux-x86_64-2.7/pyrocko
copying src/crust2x2.py -> build/lib.linux-x86_64-2.7/pyrocko
copying src/sac.py -> build/lib.linux-x86_64-2.7/pyrocko
copying src/spit.py -> build/lib.linux-x86_64-2.7/pyrocko
copying src/autopick.py -> build/lib.linux-x86_64-2.7/pyrocko
copying src/seisan_response.py -> build/lib.linux-x86_64-2.7/pyrocko
copying src/hamster_pile.py -> build/lib.linux-x86_64-2.7/pyrocko
copying src/cake.py -> build/lib.linux-x86_64-2.7/pyrocko
copying src/automap.py -> build/lib.linux-x86_64-2.7/pyrocko
copying src/quakeml.py -> build/lib.linux-x86_64-2.7/pyrocko
copying src/catalog.py -> build/lib.linux-x86_64-2.7/pyrocko
copying src/gmtpy.py -> build/lib.linux-x86_64-2.7/pyrocko
copying src/marker_editor.py -> build/lib.linux-x86_64-2.7/pyrocko
copying src/gshhg.py -> build/lib.linux-x86_64-2.7/pyrocko
copying src/moment_tensor.py -> build/lib.linux-x86_64-2.7/pyrocko
copying src/gui_util.py -> build/lib.linux-x86_64-2.7/pyrocko
copying src/parstack.py -> build/lib.linux-x86_64-2.7/pyrocko
copying src/edl.py -> build/lib.linux-x86_64-2.7/pyrocko
copying src/weeding.py -> build/lib.linux-x86_64-2.7/pyrocko
copying src/eventdata.py -> build/lib.linux-x86_64-2.7/pyrocko
copying src/config.py -> build/lib.linux-x86_64-2.7/pyrocko
creating build/lib.linux-x86_64-2.7/pyrocko/snufflings
copying src/snufflings/rms.py -> build/lib.linux-x86_64-2.7/pyrocko/snufflings
copying src/snufflings/stalta.py -> build/lib.linux-x86_64-2.7/pyrocko/snufflings
copying src/snufflings/minmax.py -> build/lib.linux-x86_64-2.7/pyrocko/snufflings
copying src/snufflings/__init__.py -> build/lib.linux-x86_64-2.7/pyrocko/snufflings
copying src/snufflings/geofon.py -> build/lib.linux-x86_64-2.7/pyrocko/snufflings
copying src/snufflings/ampspec.py -> build/lib.linux-x86_64-2.7/pyrocko/snufflings
copying src/snufflings/catalogs.py -> build/lib.linux-x86_64-2.7/pyrocko/snufflings
copying src/snufflings/download.py -> build/lib.linux-x86_64-2.7/pyrocko/snufflings
creating build/lib.linux-x86_64-2.7/pyrocko/gf
copying src/gf/targets.py -> build/lib.linux-x86_64-2.7/pyrocko/gf
copying src/gf/store.py -> build/lib.linux-x86_64-2.7/pyrocko/gf
copying src/gf/meta.py -> build/lib.linux-x86_64-2.7/pyrocko/gf
copying src/gf/builder.py -> build/lib.linux-x86_64-2.7/pyrocko/gf
copying src/gf/__init__.py -> build/lib.linux-x86_64-2.7/pyrocko/gf
copying src/gf/server.py -> build/lib.linux-x86_64-2.7/pyrocko/gf
copying src/gf/seismosizer.py -> build/lib.linux-x86_64-2.7/pyrocko/gf
copying src/gf/ws.py -> build/lib.linux-x86_64-2.7/pyrocko/gf
creating build/lib.linux-x86_64-2.7/pyrocko/fomosto
copying src/fomosto/psgrn_pscmp.py -> build/lib.linux-x86_64-2.7/pyrocko/fomosto
copying src/fomosto/qseis.py -> build/lib.linux-x86_64-2.7/pyrocko/fomosto
copying src/fomosto/ahfullgreen.py -> build/lib.linux-x86_64-2.7/pyrocko/fomosto
copying src/fomosto/__init__.py -> build/lib.linux-x86_64-2.7/pyrocko/fomosto
copying src/fomosto/qssp.py -> build/lib.linux-x86_64-2.7/pyrocko/fomosto
copying src/fomosto/dummy.py -> build/lib.linux-x86_64-2.7/pyrocko/fomosto
copying src/fomosto/qseis2d.py -> build/lib.linux-x86_64-2.7/pyrocko/fomosto
copying src/fomosto/poel.py -> build/lib.linux-x86_64-2.7/pyrocko/fomosto
creating build/lib.linux-x86_64-2.7/pyrocko/fdsn
copying src/fdsn/station.py -> build/lib.linux-x86_64-2.7/pyrocko/fdsn
copying src/fdsn/__init__.py -> build/lib.linux-x86_64-2.7/pyrocko/fdsn
copying src/fdsn/resp.py -> build/lib.linux-x86_64-2.7/pyrocko/fdsn
copying src/fdsn/enhanced_sacpz.py -> build/lib.linux-x86_64-2.7/pyrocko/fdsn
copying src/fdsn/ws.py -> build/lib.linux-x86_64-2.7/pyrocko/fdsn
creating build/lib.linux-x86_64-2.7/pyrocko/topo
copying src/topo/tile.py -> build/lib.linux-x86_64-2.7/pyrocko/topo
copying src/topo/etopo1.py -> build/lib.linux-x86_64-2.7/pyrocko/topo
copying src/topo/srtmgl3.py -> build/lib.linux-x86_64-2.7/pyrocko/topo
copying src/topo/__init__.py -> build/lib.linux-x86_64-2.7/pyrocko/topo
copying src/topo/dataset.py -> build/lib.linux-x86_64-2.7/pyrocko/topo
creating build/lib.linux-x86_64-2.7/pyrocko/fomosto_report
copying src/fomosto_report/report_call.py -> build/lib.linux-x86_64-2.7/pyrocko/fomosto_report
copying src/fomosto_report/__init__.py -> build/lib.linux-x86_64-2.7/pyrocko/fomosto_report
copying src/fomosto_report/report_main.py -> build/lib.linux-x86_64-2.7/pyrocko/fomosto_report
creating build/lib.linux-x86_64-2.7/pyrocko/data
copying src/data/snuffler.png -> build/lib.linux-x86_64-2.7/pyrocko/data
copying src/data/snuffler_help.html -> build/lib.linux-x86_64-2.7/pyrocko/data
copying src/data/snuffler_about.html -> build/lib.linux-x86_64-2.7/pyrocko/data
copying src/data/snuffler_help_epilog.html -> build/lib.linux-x86_64-2.7/pyrocko/data
creating build/lib.linux-x86_64-2.7/pyrocko/data/earthmodels
copying src/data/earthmodels/ak135-f-average.m.nd -> build/lib.linux-x86_64-2.7/pyrocko/data/earthmodels
copying src/data/earthmodels/ak135-f-average-no-ocean.f.nd -> build/lib.linux-x86_64-2.7/pyrocko/data/earthmodels
copying src/data/earthmodels/prem-no-ocean.l.nd -> build/lib.linux-x86_64-2.7/pyrocko/data/earthmodels
copying src/data/earthmodels/prem.l.nd -> build/lib.linux-x86_64-2.7/pyrocko/data/earthmodels
copying src/data/earthmodels/ak135-f-continental.m.nd -> build/lib.linux-x86_64-2.7/pyrocko/data/earthmodels
copying src/data/earthmodels/ak135-f-average-no-ocean.m.nd -> build/lib.linux-x86_64-2.7/pyrocko/data/earthmodels
copying src/data/earthmodels/ak135-f-average-no-ocean.l.nd -> build/lib.linux-x86_64-2.7/pyrocko/data/earthmodels
copying src/data/earthmodels/ak135-f-average.l.nd -> build/lib.linux-x86_64-2.7/pyrocko/data/earthmodels
copying src/data/earthmodels/prem.f.nd -> build/lib.linux-x86_64-2.7/pyrocko/data/earthmodels
copying src/data/earthmodels/prem-no-ocean.m.nd -> build/lib.linux-x86_64-2.7/pyrocko/data/earthmodels
copying src/data/earthmodels/ak135-f-average.f.nd -> build/lib.linux-x86_64-2.7/pyrocko/data/earthmodels
copying src/data/earthmodels/ak135-f-continental.l.nd -> build/lib.linux-x86_64-2.7/pyrocko/data/earthmodels
copying src/data/earthmodels/prem-no-ocean.f.nd -> build/lib.linux-x86_64-2.7/pyrocko/data/earthmodels
copying src/data/earthmodels/prem.m.nd -> build/lib.linux-x86_64-2.7/pyrocko/data/earthmodels
copying src/data/earthmodels/ak135-f-continental.f.nd -> build/lib.linux-x86_64-2.7/pyrocko/data/earthmodels
creating build/lib.linux-x86_64-2.7/pyrocko/data/colortables
copying src/data/colortables/light_sea_land.cpt -> build/lib.linux-x86_64-2.7/pyrocko/data/colortables
copying src/data/colortables/light_land_uniform.cpt -> build/lib.linux-x86_64-2.7/pyrocko/data/colortables
copying src/data/colortables/light_sea_uniform.cpt -> build/lib.linux-x86_64-2.7/pyrocko/data/colortables
copying src/data/colortables/light_land.cpt -> build/lib.linux-x86_64-2.7/pyrocko/data/colortables
copying src/data/colortables/light_sea.cpt -> build/lib.linux-x86_64-2.7/pyrocko/data/colortables
copying src/data/colortables/white_sea_land.cpt -> build/lib.linux-x86_64-2.7/pyrocko/data/colortables
creating build/lib.linux-x86_64-2.7/pyrocko/data/tectonics
copying src/data/tectonics/bird2003_plates.txt -> build/lib.linux-x86_64-2.7/pyrocko/data/tectonics
copying src/data/tectonics/gsrm1_plates.txt -> build/lib.linux-x86_64-2.7/pyrocko/data/tectonics
creating build/lib.linux-x86_64-2.7/pyrocko/data/fomosto_report
copying src/data/fomosto_report/gfreport.html -> build/lib.linux-x86_64-2.7/pyrocko/data/fomosto_report
copying src/data/fomosto_report/gfreport.tex -> build/lib.linux-x86_64-2.7/pyrocko/data/fomosto_report
running build_ext
building 'util_ext' extension
creating build/temp.linux-x86_64-2.7
creating build/temp.linux-x86_64-2.7/src
gcc -pthread -fno-strict-aliasing -g -O2 -DNDEBUG -g -fwrapv -O3 -Wall -Wstrict-prototypes -fPIC -I/home/earth/anaconda2/include/python2.7 -c src/util_ext.c -o build/temp.linux-x86_64-2.7/src/util_ext.o -Wextra
gcc -pthread -shared -L/home/earth/anaconda2/lib -Wl,-rpath=/home/earth/anaconda2/lib,--no-as-needed build/temp.linux-x86_64-2.7/src/util_ext.o -L/home/earth/anaconda2/lib -lpython2.7 -o build/lib.linux-x86_64-2.7/pyrocko/util_ext.so
building 'signal_ext' extension
gcc -pthread -fno-strict-aliasing -g -O2 -DNDEBUG -g -fwrapv -O3 -Wall -Wstrict-prototypes -fPIC -I/home/earth/anaconda2/lib/python2.7/site-packages/numpy/core/include -I/home/earth/anaconda2/include/python2.7 -c src/signal_ext.c -o build/temp.linux-x86_64-2.7/src/signal_ext.o -Wextra
gcc -pthread -shared -L/home/earth/anaconda2/lib -Wl,-rpath=/home/earth/anaconda2/lib,--no-as-needed build/temp.linux-x86_64-2.7/src/signal_ext.o -L/home/earth/anaconda2/lib -lpython2.7 -o build/lib.linux-x86_64-2.7/pyrocko/signal_ext.so
building 'mseed_ext' extension
gcc -pthread -fno-strict-aliasing -g -O2 -DNDEBUG -g -fwrapv -O3 -Wall -Wstrict-prototypes -fPIC -I/home/earth/anaconda2/lib/python2.7/site-packages/numpy/core/include -Ilibmseed -I/home/earth/anaconda2/include/python2.7 -c src/mseed_ext.c -o build/temp.linux-x86_64-2.7/src/mseed_ext.o -Wextra
In file included from src/mseed_ext.c:12:
libmseed/libmseed.h:669: warning: function declaration isn’t a prototype
libmseed/libmseed.h:686: warning: function declaration isn’t a prototype
libmseed/libmseed.h:688: warning: function declaration isn’t a prototype
gcc -pthread -shared -L/home/earth/anaconda2/lib -Wl,-rpath=/home/earth/anaconda2/lib,--no-as-needed build/temp.linux-x86_64-2.7/src/mseed_ext.o -Llibmseed -L/home/earth/anaconda2/lib -lmseed -lpython2.7 -o build/lib.linux-x86_64-2.7/pyrocko/mseed_ext.so
building 'evalresp_ext' extension
gcc -pthread -fno-strict-aliasing -g -O2 -DNDEBUG -g -fwrapv -O3 -Wall -Wstrict-prototypes -fPIC -I/home/earth/anaconda2/lib/python2.7/site-packages/numpy/core/include -Ievalresp-3.3.0/include -I/home/earth/anaconda2/include/python2.7 -c src/evalresp_ext.c -o build/temp.linux-x86_64-2.7/src/evalresp_ext.o -Wextra
gcc -pthread -shared -L/home/earth/anaconda2/lib -Wl,-rpath=/home/earth/anaconda2/lib,--no-as-needed build/temp.linux-x86_64-2.7/src/evalresp_ext.o -Levalresp-3.3.0/lib -L/home/earth/anaconda2/lib -levresp -lpython2.7 -o build/lib.linux-x86_64-2.7/pyrocko/evalresp_ext.so
building 'ims_ext' extension
gcc -pthread -fno-strict-aliasing -g -O2 -DNDEBUG -g -fwrapv -O3 -Wall -Wstrict-prototypes -fPIC -I/home/earth/anaconda2/lib/python2.7/site-packages/numpy/core/include -I/home/earth/anaconda2/include/python2.7 -c src/ims_ext.c -o build/temp.linux-x86_64-2.7/src/ims_ext.o -Wextra
gcc -pthread -shared -L/home/earth/anaconda2/lib -Wl,-rpath=/home/earth/anaconda2/lib,--no-as-needed build/temp.linux-x86_64-2.7/src/ims_ext.o -L/home/earth/anaconda2/lib -lpython2.7 -o build/lib.linux-x86_64-2.7/pyrocko/ims_ext.so
building 'datacube_ext' extension
gcc -pthread -fno-strict-aliasing -g -O2 -DNDEBUG -g -fwrapv -O3 -Wall -Wstrict-prototypes -fPIC -I/home/earth/anaconda2/lib/python2.7/site-packages/numpy/core/include -I/home/earth/anaconda2/include/python2.7 -c src/datacube_ext.c -o build/temp.linux-x86_64-2.7/src/datacube_ext.o -Wextra
gcc -pthread -shared -L/home/earth/anaconda2/lib -Wl,-rpath=/home/earth/anaconda2/lib,--no-as-needed build/temp.linux-x86_64-2.7/src/datacube_ext.o -L/home/earth/anaconda2/lib -lpython2.7 -o build/lib.linux-x86_64-2.7/pyrocko/datacube_ext.so
building 'autopick_ext' extension
gcc -pthread -fno-strict-aliasing -g -O2 -DNDEBUG -g -fwrapv -O3 -Wall -Wstrict-prototypes -fPIC -I/home/earth/anaconda2/lib/python2.7/site-packages/numpy/core/include -I/home/earth/anaconda2/include/python2.7 -c src/autopick_ext.c -o build/temp.linux-x86_64-2.7/src/autopick_ext.o -Wextra
gcc -pthread -shared -L/home/earth/anaconda2/lib -Wl,-rpath=/home/earth/anaconda2/lib,--no-as-needed build/temp.linux-x86_64-2.7/src/autopick_ext.o -L/home/earth/anaconda2/lib -lpython2.7 -o build/lib.linux-x86_64-2.7/pyrocko/autopick_ext.so
building 'gf.store_ext' extension
creating build/temp.linux-x86_64-2.7/src/gf
creating build/temp.linux-x86_64-2.7/src/gf/ext
gcc -pthread -fno-strict-aliasing -g -O2 -DNDEBUG -g -fwrapv -O3 -Wall -Wstrict-prototypes -fPIC -I/home/earth/anaconda2/lib/python2.7/site-packages/numpy/core/include -I/home/earth/anaconda2/include/python2.7 -c src/gf/ext/store_ext.c -o build/temp.linux-x86_64-2.7/src/gf/ext/store_ext.o -D_FILE_OFFSET_BITS=64 -Wextra -fopenmp
src/gf/ext/store_ext.c:157: error: redefinition of typedef ‘irecord_function_t’
src/gf/ext/store_ext.c:151: note: previous declaration of ‘irecord_function_t’ was here
src/gf/ext/store_ext.c:160: error: redefinition of typedef ‘vicinity_function_t’
src/gf/ext/store_ext.c:154: note: previous declaration of ‘vicinity_function_t’ was here
error: command 'gcc' failed with exit status 1
```

**Comment by emolch** [comment]
fixed in 1d932461666130e3478e080c5f35989a0767c409

**Comment by emolch** [comment]
fixed also in 954c880b82b961ee29b7ccb1b13f87bc53dadaa4 on `python3-testing` branch
